lib/graphql/utils.ex

```diff
@@ -113,4 +124,8 @@ defmodule BridgeEx.Graphql.Utils do
     |> LanguageConventions.to_internal_name(:read)
     |> String.to_atom()
   end
+
+  # https://elixirforum.com/t/creating-list-adding-elements-on-specific-conditions/6295/4?u=learts
+  defp append_if(list, false, _), do: list
+  defp append_if(list, true, value), do: list ++ [value]
```

Comment:
Is order of metadata important?
Cause (even though in this case it won't cause any problem) AFAIK when possible [it's more idiomatic](https://hexdocs.pm/elixir/1.13/Kernel.html#++/2) to prepend data to a list (ie. `[value | list]`) instead of appending it, since we are dealing with linked lists.